papers/eutxo/llncs.cls

```diff
@@ -0,0 +1 @@
+llncs2e/llncs.cls
```

Comment:
Can we just include `llncs.cls` by itself at this location, and leave out the rest of the `llncs2e` stuff?